client/layout/masterbar/logged-in.jsx

```diff
@@ -300,6 +302,65 @@ class MasterbarLoggedIn extends Component {
 		);
 	}
 
+	renderUpdatesMenu() {
+		const { adminMenu } = this.props;
+		if ( ! adminMenu ) {
+			return null;
+		}
+
+		let updatesCount = 0;
+		let updatesUrl = '';
+		for ( const menu of adminMenu ) {
+			for ( const menuItem of menu.children || [] ) {
+				if ( menuItem.slug === 'update-core-php' ) {
+					updatesCount = menuItem.count;
+					updatesUrl = menuItem.url;
+					break;
+				}
+			}
+		}
+
+		if ( updatesCount ) {
+			return (
+				<Item
+					className="masterbar__item-updates"
+					url={ updatesUrl }
+					icon={ <span className="dashicons-before dashicons-update" /> }
+				>
+					{ updatesCount }
+				</Item>
+			);
+		}
+		return null;
+	}
+
+	renderCommentsMenu() {
+		const { adminMenu } = this.props;
+		if ( ! adminMenu ) {
+			return null;
+		}
+
+		let commentsCount = 0;
+		let commentsUrl = '';
+		for ( const menu of adminMenu ) {
+			if ( menu.icon === 'dashicons-admin-comments' ) {
```

Comment:
I'm curious about why we use icon instead of the slug here to determine whether the menu is the comments menu or not 🤔

Reply:
The slug for sites with Default admin interface style looks like this 🙈 

<img width="725" alt="image" src="https://github.com/user-attachments/assets/156b1c87-4f82-4d08-8092-d0fe8e81fc25" />

Reply:
😱